### ISSUE TITLE
Nyquist Plug-ins: Text after control is not read by screen reader

### ISSUE THREAD

**Issue by petersampsonaudacity** [audacity/audacity#1564]
**Describe the bug**
Nyquist Plug-ins: Text after control is not read by screen reader.

Having the focus behave as expected is important for keyboard users. For them, it's not a minor issue.

Originally logged on Bugzilla as P3 2336

**To Reproduce**
1. Run any plug-in that has "Text right" while using a screen reader.
2. Observe:  Note that the "Text right" is not read.

**Expected behavior**
Text after control should be read by screen reader.

**Screenshots**
None

**Additional information (please complete the following information):**
 - OS: all OS
 - Version Audacity all up to and including 3.0.4

**Additional context**
Steve wrote on Bugzilla:
>Typical controls in Nyquist plug-ins are in the form:
;control var "Text left" <type> "Text right" <default> <min> <max>
Text supplied for "Text right" is not read by screen readers.
This bug has been present for a very long time.

**Comment by LWinterberg** [comment]
Hello! We just [switched development to Audacity 4](https://forum.audacityteam.org/t/master-has-been-switched-to-4-0-0-development/120444). We've redone, or plan on redoing, various things from scratch, including the entire UI, and the part this bug is about. As such, I'm closing this bug as it will be fixed there and we won't be fixing it for the final releases of Audacity 3.
